process/smartContract/errors.go

```diff
@@ -0,0 +1,8 @@
+package smartContract
+
+import (
+	"errors"
+)
+
+// ErrUpgradeNotAllowed signals that upgrade is not allowed
+var ErrUpgradeNotAllowed = errors.New("upgrade not allowed")
```

Comment:
more this error to process/errors.go

Reply:
upgrade is allowed only by user ?

Reply:
Fixed to "upgrade is allowed only for owner".